math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -715,6 +715,42 @@ double Mean(const RVec<T> &v)
    return double(Sum(v)) / v.size();
 }
 
+// Get the position of the greatest element of an RVec
+// In case of multiple occurrences of the maximum values,
+// the indices corresponding to the first occurrence are returned.
+///
+/// Example code, at the ROOT prompt:
+/// ~~~~{.cpp}
+/// using namespace ROOT::VecOps;
+/// RVec<float> v {1.f, 2.f, 4.f};
+/// auto v_argmax = ArgMax(v);
+/// v_argmax
+/// // (int) 2
+/// ~~~~
+template <typename T>
+long ArgMax(const RVec<T> &v)
```

Comment:
Should the return type be `Long64_t` ?

Reply:
It should probably be whatever `operator[]` takes as input.`